BlazorBarcodeScanner.ZXing.JS/BarcodeReaderInterop.cs

```diff
@@ -85,9 +85,9 @@ public async Task ToggleTorch()
             await jSRuntime.InvokeVoidAsync("BlazorBarcodeScanner.toggleTorch");
         }
 
-        public async Task<string> Capture(ElementReference canvas)
+        public async Task<string> Capture(ElementReference canvas, ElementReference video)

```

Comment:
Please split this into another PR, too many changes here, this may need a major version change.